bundles/org.openhab.binding.awattar/src/main/java/org/openhab/binding/awattar/internal/api/AwattarApi.java

```diff
@@ -141,6 +144,18 @@ public SortedSet<AwattarPrice> getData() throws AwattarApiException {
                     double netMarket = d.marketprice / 10.0;
                     double grossMarket = netMarket * vatFactor;
                     double netTotal = netMarket + basePrice;
+
+                    // in Austria the fee is added as absolute value
+                    if ("AT".equals(country)) {
+                        // add 3% absolute fee for the aWATTar service (Ausgleichskomponente)
+                        netTotal += Math.abs(netTotal) * 0.03;
+                    } else if ("DE".equals(country)) {
+                        // add 3% fee for the aWATTar service (Ausgleichskomponente)
+                        netTotal = netTotal * 1.03;
```

Comment:
AT and DE both add 3%, why are you using two different ways of doing the same?

Reply:
It was not clear ro me if they are really doing the same.
I mentioned this in the description and added a commet if someone can confirm.

I would find it strange if they would not use the absolute value to calculate the 3% fees, but i wasn't sure.

Reply:
We share the same view on this. Hopefully somebody from DE can confirm.

Reply:
I am no longer an aWATTar customer, so I can't tell finally, sorry. But however the logic looks like, I recommend to make the percentage configurable for the binding, similar to the VAT. Otherwise, if aWATTar changes the factor from 3% to 4% a new version of the binding would be necessary.

Reply:
Is there any reason to add it at all then, rather than using a transformation? For VAT, there is [VAT Transformation Service](https://www.openhab.org/addons/transformations/vat/). This seems like a simple case of multiplying a value?

Reply:
Probably it can be made this way.
The only thing that's missing would be to get an absolute value.

Reply:
Also I think adding the VAT on top of a negative price leads to wrong conception. 

The vat will be calculated over the whole monthly bill, not the individual values.